### PR TITLE
Add 'getting started' guide

### PR DESCRIPTION
**Pull request by Noviny**
Some comments from meeting to discuss this guide that sort of road-maps what I am doing:

# Decisions to validate
1. I want to use an external library to show how to add external packages, and do installs. I'm planning to use [cfonts  -  npm](https://www.npmjs.com/package/cfonts) to do this
2. Order of learning:
	1. Point to expected prior knowledge, set expectations
	2. Initialising your repository, adding the root package.json and the `workspaces` field
	3. Adding a simple `package` that exports a specific use of cfonts in `index.js`
	4. Add `placeholder-website` that cheekily uses the `simple-package`
	5. Set up building packages
		1. Install preconstruct
		2. Add `preconstruct` field in the root `package.json`
		3. Rename the `index.js` to `src/index.js`
		4. Add and set up multi-entrypoint package
			1. use this as teachable moment about when two packages depend on the same external dep (hoisting the dep)
			2. Use this as a teachable moment for `manypkg exec` to learn how to remove all the `node_modules` folders
		5. Add and set up an `app` that imports and uses the multi-entrypoint package

### CODE REVIEW

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+Finally, the repository we are building is based on [our monorepo starter](https://github.com/thinkmill/monorepo-starter), so if you simply want to be up and running, feel free to check that out.
+
+We will be assuming that you have [git](https://www.atlassian.com/git/tutorials/what-is-git) set up, as well as an [npm account](http://npmjs.com/). We are also using [yarn classic](https://classic.yarnpkg.com/lang/en/) for our package installs, so expect it to be installed globally. We are also assuming you are familiar with running [terminal commands](https://jes.st/the-terminal-101/)
+
+## Initialising your repository
+
+Firstly, we're going to need a repository to build our monorepo in. Running the following commands in your terminal will get you a new repository ready for work:
+
+```shell
```
Comment (elisechant) [inline GETTING_STARTED.md]: Isn't this opposite to what we are telling people to do in the monorepo-starter? Which is click "Use this template".

File: GETTING_STARTED.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+mkdir packages
+mkdir packages/simple-package
+```
+
+In our newly created `simple-package` directory, add the following `package.json`:
+
+```json
+{
+  "name": "@monorepo-starter/simple-package",
```
Comment (elisechant) [inline GETTING_STARTED.md]: I would like if you explain the `@` syntax

File: GETTING_STARTED.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+```json
+"scripts": {
+"postinstall": "preconstruct dev",
+"build": "preconstruct build",
+}
+```
+
+> TODO teach preconstruct fix also
+
+## Adding Manypkg to help validate your packages
```
Comment (elisechant) [inline GETTING_STARTED.md]: But why? what does "help validate your packages" mean? 

-> help validate that your packages are installed correctly 
or some such, instead?

File: GETTING_STARTED.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+yarn manypkg check
+yarn manypkg fix
+```
+
+> Add check to postinstall, updating the script above
+> Build in something that needs fixing
+> Demonstrate manypkg exec as well
+> What's the best simple example of a command we want to exec?
+
+## Setting up a publishing workflow
```
Comment (elisechant) [inline GETTING_STARTED.md]: Does publishing workflow also mean release to npm? be explicit

File: GETTING_STARTED.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
+# Getting Started with Monorepos
+
+Hi! Welcome. This guide is a very quick guide to help you set up your own monorepo, a repository that includes multiple javascript packages, all managed by a unifying root config. It aligns with our [monorepo style guide](./README.md), but is designed to walk you through the basics. If you want to have a better understanding of what decisions are being made for you, we recommend reading the guie after you've followed this tutorial.
+
+Finally, the repository we are building is based on [our monorepo starter](https://github.com/thinkmill/monorepo-starter), so if you simply want to be up and running, feel free to check that out.
+
+We will be assuming that you have [git](https://www.atlassian.com/git/tutorials/what-is-git) set up, as well as an [npm account](http://npmjs.com/). We are also using [yarn classic](https://classic.yarnpkg.com/lang/en/) for our package installs, so expect it to be installed globally. We are also assuming you are familiar with running [terminal commands](https://jes.st/the-terminal-101/)
```
[the remark this answers is not in the document]
Comment (Noviny) [reply GETTING_STARTED.md]: Probably should be a list.

This should be followable outside of TM (and by new TM people not yet versed in our lore)

File: GETTING_STARTED.md (hunk 1 above)
Comment (Noviny) [reply GETTING_STARTED.md]: This guide is how to build the starter, to teach you about monorepos. It's a getting started tutorial

File: GETTING_STARTED.md (hunk 3 above)
Comment (Noviny) [reply GETTING_STARTED.md]: We don't have a single sentence 'why you need manypkg' as the answer is basically 'there is a host of incredible pain that manypkg wards off that we hope you never need to understand'. Working on it.

File: GETTING_STARTED.md (hunk 2 above)
Comment (Noviny) [reply GETTING_STARTED.md]: It's pseudo-covered by [this part of the style guide](https://github.com/thinkmill/monorepo#one-scope-for-the-monorepo-all-packages-are-scoped) but I have made an assumption that everyone knows and understands what a scoped package is when they see it. Do you think this is not true?

File: GETTING_STARTED.md (hunk 4 above)
Comment (Noviny) [reply GETTING_STARTED.md]: I sort of assumed in an npm package set 'publishing' must mean 'publishing to npm' happy to change.

**Pull request merged**